src/constants/enum.ts

```diff
@@ -0,0 +1,10 @@
+export enum SearchResultStatus {
+  'PENDING' = 1,
+  'SUCCESS' = 2,
+  'FAILED' = 3,
+}
+
+export enum QueueNames {
```

Comment:
Minor comment but this enum naming is confusing. One value is for the worker queue itself as the other one is for the job. How about:

```suggestion
export enum AsyncProcess {
   'JOB' = 'scrapeJob',
   'QUEUE' = 'scraper',
}
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/58/commits/0a59ba3636304d6fd58950850a030ae4112dd81d